deepchem/feat/graph_data.py

```diff
@@ -204,6 +205,48 @@ def to_dgl_graph(self, self_loop: bool = False):
 
         return g
 
+    def numpy_to_torch(self):
```

Comment:
You might need to rebase this PR or start on a fresh branch since I think you have some cruft from the other PR